src/config.rs

```diff
@@ -0,0 +1,83 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use rocket::config::{Config, Environment, Limits};
+
+use dirs;
+
+#[derive(Serialize,Deserialize)]
+pub struct AWConfig {
+    #[serde(default = "default_address")]
+    address: String,
```

Comment:
Maybe we should have a section for address and port though? What should we call it?

Reply:
I don't think it's needed tbh, works fine in the top-level.